core/commands/tar.go

```diff
@@ -13,7 +13,7 @@ import (
 
 var TarCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Utility functions for tar files in IPFS.",
+		Tagline: "Interact with tar files in IPFS.",
```

Comment:
Same here: ipfs tar is a set of functions, and can't manipulate. Declarative syntax is fine.